frontend/playground-frontend/src/components/Home/Results.js

```diff
@@ -277,24 +277,33 @@ const Results = (props) => {
           <TrainVTestAccuracy />
           <TrainVTestLoss />
           {problemType.value === "classification" &&
-          auc_roc_data_res.length !== 0 ? (
+          auc_roc_data_res.length !== 0 &&
+          props.simplified === false ? (
             <AUC_ROC_curves />
           ) : null}
           {problemType.value === "classification" &&
-          auc_roc_data_res.length === 0 ? (
+          auc_roc_data_res.length === 0 &&
+          props.simplified === false ? (
             <p style={{ textAlign: "center" }}>
               No AUC/ROC curve could be generated. If this is not intended,
               check that shuffle is set to true to produce a more balanced
               train/test split which would enable correct AUC score calculation
             </p>
           ) : null}
-          {problemType.value === "classification" ? <ConfusionMatrix /> : null}
+          {problemType.value === "classification" &&
+          props.simplified === false ? (
+            <ConfusionMatrix />
+          ) : null}
         </div>
       )}
     </>
   );
 };
 
+Results.defaultProps = {
```

Comment:
why is defaultProps here needed? @farisdurrani

Reply:
We don't need this anymore. We want to remove all propTypes and initializing props outside of the arguments